tests/ssg_test_suite/rule.py

```diff
@@ -391,8 +391,10 @@ def _load_all_tests(self, rule):
         product_yaml = common.get_product_context(self.test_env.product)
         # Initialize a mock template_builder.
         empty = "/ssgts/empty/placeholder"
+        product_build_dir = os.path.join(common.SSG_ROOT, "build", product_yaml["product"])
         template_builder = ssg.templates.Builder(
-            product_yaml, empty, common._SHARED_TEMPLATES, empty, empty)
+            product_yaml, empty, common._SHARED_TEMPLATES, empty, empty,
+            product_build_dir + "/platforms", product_build_dir + "/cpe_items")
```

Comment:
We probably should have a constant in `ssg.common` defining `platforms` and `cpe_items`